bitnami/airflow/values.yaml

```diff
@@ -970,6 +970,14 @@ metrics:
   ##
   hostAliases: []
 
+  ## Prometheus Operator ServiceMonitor resource
+  serviceMonitor:
+    enabled: false
+    interval: 60s
+    scrapeTimeout: 10s
+    # ServiceMonitor Labels
+    labels: {}
+
```

Comment:
sorry I missed this. Could you add the namespace in values.yaml?
```
## Namespace in which Prometheus is running
    ##
    # namespace: monitoring
    ## Interval at which metrics should be scraped.
    ## ref: https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#endpoint
```

Reply:
@Mauraza Can you check now?

Reply:
@Mauraza can you check now?